src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -834,6 +834,20 @@ if (typeof Sfjs === 'undefined' || typeof Sfjs.loadToolbar === 'undefined') {
                 });
             },
 
+            initializeMailerTable: function() {
```

Comment:
Woudln't it be possible to avoid having all the JS code specific to panels in base_js.html.twig, which is the JS code shared between the toolbar and the profiler UI ? This currently increases the size of the toolbar.

Reply:
A solution is to have this code in the `<script>`  tag of the panel directly instead of having a script tag that calls this method.

Reply:
btw, the same should probably be done for `updateLogsTable` and maybe other methods

Reply:
OK ... but let's do that in another PR so we can extract all foreign JS code at once. Thanks